openmdao/components/tests/test_meta_model_unstructured_comp.py

```diff
@@ -929,5 +927,54 @@ def test_metamodel_deprecated(self):
         prob.run_model()
         assert_rel_error(self, prob['mm.y1'], 1.5, 1e-2)
 
+    def test_metamodel_use_fd_if_no_surrogate_linearize(self):
+        class Trig(MetaModelUnStructuredComp):
+            def setup(self):
+
+                surrogate = FloatKrigingSurrogate()
+                self.add_input('x', 0.,
+                               training_data=np.linspace(0,10,20))
+                self.add_output('sin_x', 0.,
+                                surrogate=surrogate,
+                                training_data=.5*np.sin(np.linspace(0,10,20)))
+
+                self.declare_partials(of='sin_x', wrt='x', method='fd')
```

Comment:
The `declare_partials` with method='fd' needs to be called automatically instead of making the user call it.